tests/system/fim/scenarios/212_realtime_scheduled_monitoring/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+elastic_verification_extra_arguments: "-r 3 -s 300"
```

Comment:
Maybe updating these values with `set_fact` before running the `verify_alerts.yml` in the `launch_test_escenario.yml` task is more explicit as we can see which parameters are being the tasks launched with. E.G: https://github.com/wazuh/wazuh-qa/blob/9455e81de250cfb7a9077da377925afe70ae6782/tests/system/fim/scenarios/206_tag_option/launch_test_scenario.yml#L11-L15

Reply:
https://github.com/wazuh/wazuh-qa/pull/562/commits/8a02576792131a97686733b15808c140c0ddd5b2